src/devices-tint.html

```diff
@@ -0,0 +1,101 @@
+                });
+            });
+
+            RED.bavaria.ui.addNumInput("#node-input-brightnessChange");
+            RED.bavaria.ui.addNumInput("#node-input-temperatureChange");
+        },
+        oneditsave: function() {
+            this.brightnessChange = parseFloat(this.brightnessChange);
+            this.temperatureChange = parseInt(this.temperatureChange);
+
+            console.log(this);
+        }
+    });
+
+</script>
+
+<script type="text/html" data-template-name="tint-remote">
+
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> Name</label>
+        <input type="text" id="node-input-name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-bridge"><i class="fa fa-tag"></i> Bridge</label>
+        <input type="text" id="node-input-bridge" placeholder="bridge" />
+    </div>
+    <div class="form-row">
+        <label for="node-input-deviceName"><i class="fa fa-tag"></i> Device</label>
+        <select id="node-input-deviceName"></select>
+    </div>
+    <div class="form-row">
+        <label for="node-input-suppressPowerpayload"><i class="fa fa-tag"></i> Suppress power button payload</label>
+        <input type="checkbox" id="node-input-suppressPowerpayload" />
+    </div>
+    <div class="form-row">
+        <label for="node-input-brightnessChange"><i class="fa fa-tag"></i> Brightness step</label>
+        <input type="text" id="node-input-brightnessChange">
+    </div>
+    <div class="form-row">
+        <label for="node-input-temperatureChange"><i class="fa fa-tag"></i> Temperature step</label>
+        <input type="text" id="node-input-temperatureChange">
+    </div>
+
+</script>
+
+<script type="text/html" data-help-name="tint-remote">
+    <p>Sends a new message when a Müller Licht - tint remote is pressed.</p>
+    <p>It is only compatible with Müller Licht - tint remote (Model: mli-404011)</p>
+    <h3>Output</h3>
+    <p>The output depends on the button that was pressed and some buttons behave a bit weird. The device has a bit of memory which may interfere with the logic you want to build.</p>
+    <p>All of the buttons will set <code>msg.action</code> with 2 properties.</p>
+    <ul>
+        <li><code>name</code> - name of the button</li>
+        <li><code>description</code> - the type of how the button was pressed</li>
+    </ul>
+    <h4>Power button</h4>
+    <p>The remote has a small memory that will save the power state. The state will be set to <code>msg.payload</code> You can suppress power button payload in the config. If you do, the payload is <b>toggle</b></p>
+    <h4>Color wheel</h4>
+    <p>The color is also stored on the remote itself. You can press the same spot on the wheel more than once and it will shift the color slightly to the color you pressed.</p>
+    <p>The <code>msg.payload</code> of this node will be a color-override. Simply place the lamps behind this node you want to control followed by a send-message node.</p>
```

Comment:
Place the lamps you want to control behind this node, followed by a send-message node.